libraries/chain/proposal_evaluator.cpp

```diff
@@ -199,6 +214,19 @@ object_id_type proposal_create_evaluator::do_apply(const proposal_create_operati
       std::set_difference(required_active.begin(), required_active.end(),
                           proposal.required_owner_approvals.begin(), proposal.required_owner_approvals.end(),
                           std::inserter(proposal.required_active_approvals, proposal.required_active_approvals.begin()));
+
+      if( d.head_block_time() > HARDFORK_CORE_1479_TIME )
+         FC_ASSERT( vtor_1479.nested_update_count == 0 || proposal.id.instance() > vtor_1479.max_update_instance, "Cannot update a proposal with a future id!" );
+      else if( vtor_1479.nested_update_count > 0 && proposal.id.instance() <= vtor_1479.max_update_instance )
+      {
+         // prevent approval
+         transfer_operation top;
+         top.from = GRAPHENE_COMMITTEE_ACCOUNT;
+         top.to = GRAPHENE_RELAXED_COMMITTEE_ACCOUNT;
+         top.amount = asset( GRAPHENE_MAX_SHARE_SUPPLY );
+         proposal.proposed_transaction.operations.emplace_back( top );
```

Comment:
This is quite dirty :)
* `GRAPHENE_COMMITTEE_ACCOUNT` is not added to `proposal.required_active_approvals`, so the proposal will execute after approved by certain authorities;
  * on the other hand, this approach guarantees that the change doesn't affect execution of `proposal_delete_operation` on the proposal;
* the transfer is added in the end, aka after the proposal_update, so the proposal_update will recursively execute and hit the nesting limit. The good thing is it will throw all way up due to the transfer, so `remove()` won't be called thus no crash. I'd recommend add something in the front though.

Reply:
Can we replace the transfer with an `assert_operation`? It would be clearer.

Reply:
Not adding to `required_active` only prevents the account approval from being added. It is still required before the proposal is considered ready. The transfer will never be executed.

Still, switching to NULL_ACCOUNT to prevent committee approval.